lib/database.js

```diff
@@ -37,7 +37,7 @@ exports.connect = function connect(options) {
 exports.ensureConnectionWorks = (connection) => {
     return connection.raw('SELECT 1+1 as RESULT;')
         .catch((err) => {
-            if (err.errno === 'ENOTFOUND' || err.code === 'ETIMEDOUT' || err.code === 'EAI_AGAIN') {
+            if (err.code === 'ENOTFOUND' || err.code === 'ETIMEDOUT' || err.code === 'EAI_AGAIN') {
```

Comment:
@tpatel does the `.errno` never ever happen again? What was the reason it was removed?

Reply:
The `errno` was always supposed to be a number, but for some reason before https://github.com/nodejs/node/pull/28140 it returned a string sometimes. Since node 13, this has been fixed, `errno` is always a number and `code` is always a string like `ENOTFOUND` or `ETIMEDOUT`.

Reply:
If I understand this correctly then in case of node 10.x and 12.x, which we actively support, there's a possibility to still get the `errno`? If that's true we'd probably want to keep an extra `|| err.errno ===` with a note to remove once v12 goes out of service?

Reply:
`code` will always be `ENOTFOUND`, but `errno` can be a number or `ENOTFOUND` depending on the nodejs version, so it's better to use code when comparing to a string.
If we really want to keep `errno` then we would have to do something like `err.errno === 'ENOTFOUND' || err.errno === 123`. 

You can get more information about a similar change in the ghost codebase here: https://github.com/TryGhost/Ghost/commit/ba1ffb7b5f52f629c3a146e1264e2d4aedf00971

Reply:
Anyway, this commit shouldn't probably be in here 😁

Reply:
right 🙊

Reply:
better now 🧹